### PR TITLE
handle duplicates in --include-*

### PR DESCRIPTION
**Pull request by boegel**
fix for #2150

I verified that the enhanced tests indeed replicate the bug reported in #2150.

cc @akesandgren

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/tools/include.py [hunk 1]
```diff
@@ -162,7 +162,8 @@ def include_easyblocks(tmpdir, paths):
         else:
             target_path = os.path.join(easyblocks_dir, filename)
 
-        symlink(easyblock_module, target_path)
+        if not os.path.exists(target_path):
```
Comment (wpoely86) [inline easybuild/tools/include.py]: what if it exists and points to something else? Can that happen?

Comment (boegel) [inline easybuild/tools/include.py]: no, this is in a dedicated tmpdir

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**